docs/concepts/libp2p.md

```diff
@@ -33,43 +33,43 @@ Libp2p works with a lot of different addressing schemes in a consistent way. A m
 
 The technology used to move your data from one machine to another. Transports are defined in terms of two core operations, _listening_ and _dialing_. Listening means that you can accept incoming connections from other peers. _Dialing_ is the process of opening an outgoing connection to a listening peer. One of Libp2p's core requirements is to be _transport agnostic_, meaning the decision of what transport protocol to use is up to an application's developer (who may decide to support many different _transports_ at the same time).
 
-#### [Security](https://docs.libp2p.io/introduction/#secure-communication)
+#### [Security](https://docs.libp2p.io/introduction/what-is-libp2p/#security)
```

Comment:
This should be concepts/secure-comm/overview/ instead of referring to the introductory page.